nixos/tests/postgresql.nix

```diff
@@ -24,6 +26,9 @@ let
       {
         services.postgresql.package=postgresql-package;
         services.postgresql.enable = true;
+        services.postgresql.initialScript = pkgs.writeText "init.sql" ''
+          CREATE USER root WITH SUPERUSER;
```

Comment:
maybe it would be cleaner to run ```psql``` with ```-U postgres``` ?
superuser name was changed from ```root``` to ```postgres``` some time ago, but ```psql``` defaults to the name of the current linux user  which is ```root``` in the tests